vignettes/data_from_incidence2.Rmd

```diff
@@ -0,0 +1,149 @@
+
+We first load some linelist data for the 2014 Ebola outbreak provided with the _outbreaks_ package, and create an _incidence2_ object and view it.
+
+```{r}
+# load ebola dataset from outbreak
+data("ebola_sim_clean")
+ebola <- ebola_sim_clean$linelist
+
+# view ebola
+head(ebola)
+```
+
+We then prepare an `<incidence2>` object using the `date_of_onset` and `date_of_outcome` columns as the date indices, and grouping by outcome.
+
+These steps use a combination of functions from _incidence2_ as well as common data science packages such as [_dplyr_](https://dplyr.tidyverse.org/).
+
+```{r}
+# create incidence2 object of ebola deaths
+ebola <- incidence(
+  x = ebola,
+  date_index = c(
+    onset = "date_of_onset",
+    outcome = "date_of_outcome"
+  ),
+  groups = "outcome"
+)
+
+# filter for outcomes that are deaths using dplyr::filter --- death counts
+# also filter for all onsets --- these are the case counts
+ebola <- filter(
+  ebola,
+  (outcome == "Death" & count_variable == "outcome") |
+    (count_variable == "onset")
+)
+
+# remove groups using incidence2::regroup()
+ebola <- regroup(ebola)
+
+# prepare data
+ebola <- prepare_data(
+  ebola,
+  cases_variable = "onset",
+  deaths_variable = "outcome"
+)
+```
+
+We then load a delay distribution from the _epiparameter_ database as an `<epidist>` object.
+
+```{r}
+onset_to_death_ebola <- epidist_db(
+  disease = "Ebola Virus Disease",
+  epi_dist = "onset_to_death",
+  author = "Barry_etal"
+)
+```
+
+Finally, we estimate the static case fatality rate while correcting for delays suing the `estimate_static()` function.
+
+```{r}
+# estimate static CFR as a sanity check
+estimate_static(
+  ebola,
+  correct_for_delays = TRUE, epi_dist = onset_to_death_ebola
+)
+```
+
+## `<incidence2>` objects from aggregated case data
+
+Aggregated case data such as that provided by the package [_covidregionaldata_](https://epiforecasts.io/covidregionaldata/) can be used with _datadelay_ directly.
+
+Such usage is shown in other vignettes such as the [vignette on estimating time-varying severity](estimate_time_varying_severity.html).
+
+Here we show how to transform such data into `<incidence2>` objects, and then prepare them for _datadelay_.
+
+The columns in this dataset are already in a format that can be converted into an `<incidence2>` object using `incidence2::incidence()`, and then handled by `prepare_data()`.
+The code chunks below show how this is to be done.
+
+```{r}
+# get data bundled with the {incidence2} package
+covid_uk <- covidregionaldataUK
+
+# view the data
+head(covid_uk)
+```
+
+Note that the grouping structure of this dataset given by the "region" variable is present in the `<incidence2>` object, but is overridden to provide a single dataset with cases and deaths summed across regions by date by `prepare_data()`.
+This functionality is currently under development and might change in future.
+
+```{r}
+# convert to incidence2 object
+covid_uk_incidence <- incidence(
```

Comment:
Thanks - I agree, and this is laid out in the text just before this chunk. Still, I think it's useful to include as a demonstration since it's in the {incidence2} vignette.